### PR TITLE
Add mandarin link to ReadTheDocs layout

### PR DESCRIPTION
**Pull request by Crissman**
## Motivation
Add a link for the Mandarin documentation to the ReadTheDocs page.

## Description of the changes
Add a link for the Mandarin documentation to the ReadTheDocs page.

### CODE REVIEW

File: docs/source/_templates/layout.html [hunk 1]
```diff
@@ -21,6 +21,9 @@
                 <li>
                     <a href="https://optuna.org/#paper" class="header_link">Paper</a>
                 </li>
+                <li>
+                    <a href="https://optuna-zh-cn.readthedocs.io/" class="header_link">中文文档</a>
+                </li>
```
Comment (toshihikoyanase) [inline docs/source/_templates/layout.html:26]: I think we aligned the navbar of `ReadTheDocs` with that of https://optuna.org. How about changing the link text to `Community`?

The nav bar of this PR
image

The nav bar of https://optuna.org
image



```suggestion
                <li>
                    <a href="https://optuna.org/#community" class="header_link">Community</a>
                </li>
```

**Review by toshihikoyanase — CHANGES_REQUESTED** [review]
Thank you for your PR. I agree to update the navbar, but I have a comment about the link text and URL.

**Comment by Crissman** [comment]
Agreed and done. 🙇‍♂️

**Comment by toshihikoyanase** [comment]
@Crissman Thanks! Could you merge the master branch to resolve CI errors?

**Comment by Crissman** [comment]
Master merged.

**Review by toshihikoyanase — APPROVED** [review]
Thank you for your update. LGTM!

**Pull request merged**